src/Microsoft.Health.Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json

```diff
@@ -62,7 +62,7 @@
     {"path": "FamilyMemberHistory.deceased", "method": "redact"},
     {"path": "FamilyMemberHistory.condition.onset as Age", "method": "perturb", "span": 0.1, "rangeType": "proportional", "roundTo": 0},
     {"path": "FamilyMemberHistory.condition.onset", "method": "redact"},
-    {"path": "ClinicalImpression.description", "method": "redact"},  
+    {"path": "ClinicalImpression.description", "method": "presidio"},  
```

Comment:
I would drop presidio as default in all of the configurations since it will fail in case presidio containers are not set.

Reply:
I'm not sure because the whole point of this PR is to show a POC of Presidio

Reply:
agree. Let's add a new config sample file named configuration-sample-presidio.json for reference in this PR

Reply:
In that case whoever uses the tool as of now will have to start a presidio cluster, or else  the execution it will end with failure

Reply:
How about adding a default - if presidio cluster is N/A - do a basic redact? (or the CLI)

Reply:
I like the separate config file, it won't fail (my comment was to Itye)

Reply:
@SharonHart could be a good workaround for now, and for future implemetation add a default anonymization in case presidio is not configured